tests/torchtune/datasets/test_chat_dataset.py

```diff
@@ -146,15 +147,18 @@ def test_get_item(self, mock_load_dataset, template, dialogue):
                 4,
                 2,
                 3,
+                -1,
+                0,
                 5,
                 6,
                 11,
                 10,
                 1,
+                6,
                 -1,
             ]
         ]
-        prompt_lengths = (14, 4)
+        prompt_lengths = (15, 5)
```

Comment:
Why is stuff in chat dataset changing, just cause of dummy tokenizer update? Also why is it not incremented by two if you're adding both bos and eos? (I'm prob missing something obvious here)

Reply:
yes it's the dummy tokenizer update. the expected prompt and label are actually two turns, so going from (14, 4) -> (15, 5) is actually incrementing by 2